runtime-modules/constitution/src/benchmarking.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::{Call, ConstitutionInfo, Event, Module, Trait};
+use frame_benchmarking::benchmarks;
+use frame_system::Module as System;
+use frame_system::{EventRecord, RawOrigin};
+use sp_runtime::traits::Hash;
+use sp_std::boxed::Box;
+use sp_std::vec;
+use sp_std::vec::Vec;
+
+fn assert_last_event<T: Trait>(generic_event: <T as Trait>::Event) {
+    let events = System::<T>::events();
+    let system_event: <T as frame_system::Trait>::Event = generic_event.into();
+    // compare to the last event record
+    let EventRecord { event, .. } = &events[events.len() - 1];
+    assert_eq!(event, &system_event);
+}
+
+benchmarks! {
+    _{ }
+
+    amend_constitution{
+        let i in 1 .. 50000;
```

Comment:
I think it would be a good idea to move the upper-bound constant(50000) into a constant `MAX_BYTES` to be consistent with the benchmark for the other pallets

Reply:
Ok.